lib/irb/command/cd.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module IRB
+  module Command
+    class CD < Base
+      category "Workspace"
+      description "Move into the given object or leave the current context."
+
+      help_message(<<~HELP)
+        Usage: cd ([target]|..)
+
+        When given an object, cd will move into that object, making it the current context.
+        When given ".." or no arguments, cd will leave the current context, moving back to the previous context.
+
+        Examples:
+
+          cd Foo
+          cd Foo.new
+          cd @ivar
+          cd ..
+          cd
+      HELP
+
+      def execute(arg)
+        case arg
+        when "..", ""
+          irb_context.pop_workspace
```

Comment:
In the terminal, `cd` (without arguments) goes to the user's home, unlike `cd ..`. Rather than combine the two, should we only support `cd ..`, and leave `cd` as a possible future feature? That was, it wouldn't be a breaking change to introduce whatever the closest behavior would be to "return home".

Reply:
Good catch, I thought in Pry it goes back a level, but it goes to the top-level instead. Will update 👍